recipe/meta.yaml

```diff
@@ -11,38 +11,42 @@ source:
   sha256: 5cac870d6d269cdd71d28def4cae310c609c02aac636c0831176188f16021f44
 
 build:
+  skip: True  # [py<37]
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script:
+    - {{ PYTHON }} build.py
```

Comment:
I don't know how poetry works but maybe you don't need to call this. It is listed in the pyproject.toml and maybe poetry will call it for you. See https://github.com/predict-idlab/plotly-resampler/blob/b1225bd05d5d93821e06cb30fa589759e768a40b/pyproject.toml#L34